source/analysis/REIXS/REIXSXESImageInterpolationAB.cpp

```diff
@@ -303,6 +303,7 @@ void REIXSXESImageInterpolationAB::setInputDataSourcesImplementation(const QList
 	cachedValues_ = QVector<double>(axes_.at(0).size);
 	axisValueCacheInvalid_ = true;
 	cachedAxisValues_ = QVector<double>(axes_.at(0).size);
+	onInputSourceSizeChanged();

```

Comment:
@dretrex, this fixes it, but I don't think it's the right way to do it. In fact, i don't think that it should be needed, since when a source is connected ```inputSource_->signalSource()/sizeChanged(int)``` is connected to ```onInputSourceSizeChanged()``` (line 297 above) and on line 309 below the ```emitSizeChanged(0)``` is called which should trigger this anyways, if things are all connected up properly, shouldn't it?